src/integration-tests/bash/cleanup.sh

```diff
@@ -321,7 +318,7 @@ if [ -x "$(command -v helm)" ]; then
      helm delete --purge  $helm_name
   done
 
-  # cleanup tiller artifacts that are created in run.sh
```

Comment:
If cleanup.sh only cleans up artifacts that are created in run.sh and run.sh is not removed, shall we remove this helper script as well? Loop in QA and see if cleanup.sh is still in use.

Reply:
The cleanup.sh is called in the java test and other test scripts.

Reply:
okay. thanks.